templates/listing-page.twig

```diff
@@ -7,11 +7,11 @@
             <h2>{{ listing_page_title ?? __( 'All articles', 'planet4-master-theme' ) }}</h2>
         {% endif %}
         {% if layout == 'list' %}
-            <button class="grid-view-toggle" title="{{__( 'Grid layout', 'planet4-master-theme' ) }}">
+            <button class="grid-view-toggle layout-toggle" title="{{__( 'Grid layout', 'planet4-master-theme' ) }}">
```

Comment:
Do we need to `layout-toggle` buttons? If we're changing its content in JS anyway, isn't it enough to have one? Also, do we still need the `grid-view` and `list-view` icons if we add the svg in JS?

Reply:
This was a rough implementation based on your current work. 

Ideally, we should have one button and we change the button background image or content based on which layout is visible at the moment. I added the class just to have a similar name to use for the button in all listing pages